src/main/kotlin/org/jitsi/nlj/dtls/DtlsClient.kt

```diff
@@ -43,7 +43,7 @@ class DtlsClient(
     fun connect(): DTLSTransport {
         try {
             return dtlsClientProtocol.connect(this.tlsClient, datagramTransport).also {
-                logger.cinfo { "DTLS handshake finished" }
+                logger.cdebug { "DTLS handshake finished" }
```

Comment:
I actually find this log very useful to be on INFO, and it isn't too verbose (once per EP).  Was there a reason to put it on DEBUG?

Reply:
Yes, there's multiple lines logged for the same event:
```
JVB 2020-02-02 18:36:17.258 INFO: [20511] [confId=xxx gid=xxx stats_id=xxx conf_name=xxx epId=xxx local_ufrag=xxx] TlsClientImpl.notifyServerVersion#167: Negotiated DTLS version DTLS 1.2
JVB 2020-02-02 18:36:17.463 INFO: [20511] [confId=xxx gid=xxx stats_id=xxx conf_name=xxx epId=xxx local_ufrag=xxx] DtlsClient.connect#46: DTLS handshake finished
JVB 2020-02-02 18:36:17.463 INFO: [20511] [confId=xxx epId=xxx local_ufrag=xxx gid=xxx stats_id=xxx conf_name=xxx] DtlsTransport.lambda$new$0#130: DTLS handshake complete. Got SRTP profile 1
```